source/assets/js/modules/profile/controllers/profile.controller.js

```diff
@@ -29,6 +29,26 @@ angular.module('locApp.modules.profile.controllers')
             $scope.languages = response;
           });
 
+          /**
+           * @ngdoc function
+           * @name checkURI
```

Comment:
this says checkURI but the function seems to be named checkURL ?

Also, this new function could get a unit test, IMO.  (if it's not too hard)

Reply:
I agree that we could use a unit test for the `checkURL()` function; since this is in the Angular controller, should we mirror the way the code is laid out in the tests directory, i.e.:

The source code file is `assets/js/modules/profile/controllers` directory with the corresponding unit tests in something like `__tests__/modules/profile/controllers/profile.controller.test.js`?